DepositeCalcTests/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using NUnit.Framework;
+using DepositeCalcTests.Pages;
+using System;
+using System.Threading;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class HistoryPageTests : BaseTest
+    {
+        private HistoryPage historyPage;
+
+        [SetUp]
+        public void Setup()
+        {
+            InitDriver("https://localhost:5001/History");
+            historyPage = new HistoryPage(driver);
+        }
+
+        [Test]
+        public void CalculatorLinkTest()
+        {
+            // Act
+            var calculatorPage = historyPage.Calculator();
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsOpened(), "Incorrect page");
+        }
+
+        [Test]
+        public void TitleTest()
+        {
+            // Assert
+            Assert.AreEqual("History", driver.Title, "Incorrect title");
+        }
+
+        [Test]
+        public void LastCalculationHistoryTest()
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var depositAmount = "123";
+            var interestRate = "100";
+            var investmentTerm = "321";
+            var expectedLastCalculationValues = new[]
+            {
+                "123.00",
+                "100",
+                "321",
+                "365",
+                "231.17",
+                "108.17",
+            };
+
+            // Act
+            driver.Url = "https://localhost:5001/Calculator";
+            calculatorPage.ClearingMandatoryTextFields();
+            calculatorPage.FinancialYear = "365";
+            calculatorPage.FillingMandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.Calculate();
+            Thread.Sleep(1000); // doesn't work without this delay
+            calculatorPage.OpenHistory();
+            historyPage.WeitForReady();
```

Comment:
It will work without sleep...
```suggestion
            calculatorPage.Calculate();
            calculatorPage.OpenHistory();
            historyPage.WeitForReady();
```

...but you should update the wait in `Calculate` method:
```
        public void Calculate()
        {
            CalculateBtn.Click();
            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => CalculateBtn.Enabled);
        }
```